src/main/resources/data.properties

```diff
@@ -55,6 +55,11 @@ actions.max=2
 # Ingest billpay
 ingest.billpay=false
 ingest.billpay.accounts=false
+# ingest pockets
+ingest.pockets=false
+pockets.min=1
+pockets.max=3
+pockets.currency=EUR
```

Comment:
please also add a property specifying the path to the pockets.json file instead of hardcoding it on the code.

Reply:
I will do that